### PR TITLE
okta-support-added-in-lineage

### PR DESCRIPTION
**Pull request by codingwithabhi**
### Describe your changes :

I have added okta support in lineage

#
### Type of change 

- [x] Improvement
- [x] New feature

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [ x ] I have performed a self-review of my own. 
- [ x ] I have tagged my reviewers below.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ x ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers

@ayush-shah

### CODE REVIEW

File: ingestion/src/airflow_provider_openmetadata/lineage/openmetadata.py [hunk 1]
```diff
@@ -51,6 +51,18 @@ def get_lineage_config() -> OpenMetadataLineageConfig:
...
+            return OpenMetadataLineageConfig.parse_obj(
+                {
+                    "airflow_service_name": airflow_service_name,
+                    "api_endpoint": api_endpoint,
+                    "auth_provider_type": auth_provider_type,
+                    "secret_key": secret_key,
+                    "token": token,
+                }
+            )
+
```
Comment (harshach) [inline ingestion/src/airflow_provider_openmetadata/lineage/openmetadata.py:65]: @codingwithabhi we can't expect users to give us token, we need to generate that token just like we are doing with ingestion with okta
also add else if auth_provider_type == "google"

Comment (harshach) [reply ingestion/src/airflow_provider_openmetadata/lineage/openmetadata.py:65]: for google we are not asking token, it generates that token

Comment (harshach) [reply ingestion/src/airflow_provider_openmetadata/lineage/openmetadata.py:65]: IMO I don't think we need to check anything here, just pass those as key/value pairs and OpenMetadataLineageConfig should parse additional details

**Comment by ayush-shah** [comment]
Stale PR, WIP in #2162.

**Pull request closed without merging**